docs/antrea-ipam.md

```diff
@@ -376,9 +376,9 @@ metadata:
 spec:
   ipVersion: 4
   ipRanges:
-  - cidr: "10.10.1.0/26"
-    gateway: "10.10.1.1"
-    prefixLength: 24
+    - cidr: "10.10.1.0/26"
```

Comment:
The original writing is also correct and widely used in K8s docs. Even [the offical example](https://yaml.org/spec/1.2.2/#21-collections) and [go-yaml ](https://github.com/go-yaml/yaml#example) are using the original format.
I think it's good as long as the indention is consistent in one declaration, and it's hard and unnecessary to strictly enforce one indention.

Reply:
updated